Jobs.Fetcher.YouTube/Helpers/ApiDataFetcher.cs

```diff
@@ -172,16 +177,21 @@ public static class ApiDataFetcher {
 
         public static List<(DateTime date, long subscriberViews)> FetchSubscriberViews(YTA.VideoDailyMetric mostRecentRecord, string channelId, YTD.Video video, DateTime now, YouTubeAnalyticsService analyticsService, Logger logger, bool reprocessMetrics = false) {
```

Comment:
Does "now" have to be passed as a parameter here? Especially since the function only uses the date, and not a high precision time